dask_geomodeling/tests/test_raster.py

```diff
@@ -1158,6 +1158,11 @@ def setUp(self):
             **self.request,
         }
 
+    def test_request(self):
+        view = self.klass(self.raster, frequency="D", statistic="sum")
+        (req, _), _, _ = view.get_sources_and_requests(**self.request_all)
+        self.assertEqual(req["frequency"], "D")
+
```

Comment:
This checks if the TemporalAggregate block propagates the `frequency` parameter. But shouldn't the AggregateRaster also pass that parameter? Otherwise it will never arrive at the TemporalAggregate when the latter is requested from the former. I think we didn't discuss this, I just thought of it now.